### PR TITLE
Configure Travis Configuration File for Android.

### PR DESCRIPTION
**Pull request by vanniktech**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
-language: java
+language: android
+
+android:
+  components:
+    - tools
+    - platform-tools
+    - build-tools-26.0.0
+    - android-26
+    - extra-android-support
```
Comment (JakeWharton) [inline .travis.yml]: This isn't needed

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
-language: java
+language: android
+
+android:
+  components:
+    - tools
+    - platform-tools
+    - build-tools-26.0.0
+    - android-26
```
Comment (JakeWharton) [inline .travis.yml]: We can have Gradle install the platform, build-tools, and m2repositories.

Comment (vanniktech) [inline .travis.yml]: Does not seem like it's working.

Comment (JakeWharton) [inline .travis.yml]: The license file need written out first.

Comment (vanniktech) [inline .travis.yml]: Any sample for this? I'm still letting travis download the SDK like this.

Comment (JakeWharton) [inline .travis.yml]: Yeah I'll update it today. Was a bit busy over the weekend, sorry.

**Pull request closed without merging**